### PR TITLE
Added 2 Examples to install-powershell.ps1 which fixes #10912

### LINKED ISSUE #10912: install-powershell.ps1 contains no Examples on how to run it
install-powershell.ps1 doesn't contain any example code on how to run it at all.

### PR DESCRIPTION
**Pull request by kilasuit**
Fixes issue #10912

# PR Summary

As noted in #10911 & #10912 there was no examples for using this script - this PR fixes that

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [x] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [x] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [x] Issue filed: 
- **Testing - New and feature**
    - [x] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
- **Tooling**
    - [x] I have considered the user experience from a tooling perspective and don't believe tooling will be impacted.
    - **OR**
    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This may include:
        - Impact on [PowerShell Editor Services](https://github.com/PowerShell/PowerShellEditorServices) which is used in the [PowerShell extension](https://github.com/PowerShell/vscode-powershell) for VSCode (which runs in a different PS Host).
        - Impact on Completions (both in the console and in editors) - one of PowerShell's most powerful features.
        - Impact on [PSScriptAnalyzer](https://github.com/PowerShell/PSScriptAnalyzer) (which provides linting & formatting in the editor extensions).
        - Impact on [EditorSyntax](https://github.com/PowerShell/EditorSyntax) (which provides syntax highlighting with in VSCode, GitHub, and many other editors).

### CODE REVIEW

File: tools/install-powershell.ps1 [hunk 1]
```diff
@@ -17,6 +17,12 @@
     On Windows, add the absolute destination path to the 'User' scope environment variable 'Path';
     On Linux, make the symlink '/usr/bin/pwsh' points to "$Destination/pwsh";
     On MacOS, make the symlink '/usr/local/bin/pwsh' points to "$Destination/pwsh".
+ .EXAMPLE
+    Install the daily build
+    .\install-powershell.ps1 -Daily
+ .EXAMPLE
+    Invoke this script directly from GitHub
+    iex "& { $(irm 'https://raw.githubusercontent.com/PowerShell/PowerShell/master/tools/install-powershell.ps1') } -daily"
```
Comment (TravisEz13) [inline tools/install-powershell.ps1]:
```suggestion
    iex "& { $(irm 'https://aka.ms/install-powershell.ps1') } -daily"
```

Comment (TravisEz13) [reply tools/install-powershell.ps1]: This allows us a little flexibility in managing the script

**Review by TravisEz13 — APPROVED** [review]

**Comment by TravisEz13** [comment]
@PoshChan Please remind me in 20 hours

**Review by iSazonov — APPROVED** [review]

File: tools/install-powershell.ps1 [hunk 2]
```diff
@@ -17,6 +17,12 @@
     On Windows, add the absolute destination path to the 'User' scope environment variable 'Path';
     On Linux, make the symlink '/usr/bin/pwsh' points to "$Destination/pwsh";
     On MacOS, make the symlink '/usr/local/bin/pwsh' points to "$Destination/pwsh".
+ .EXAMPLE
+    Install the daily build
+    .\install-powershell.ps1 -Daily
+ .EXAMPLE
+    Invoke this script directly from GitHub
+    iex "& { $(irm 'https://aka.ms/install-powershell.ps1') } -daily"
```
Comment (iSazonov) [inline tools/install-powershell.ps1]:
```suggestion
    Invoke-Expression "& { $(Invoke-RestMethod 'https://aka.ms/install-powershell.ps1') } -daily"
```

**Comment by PoshChan** [comment]
@TravisEz13, this is the reminder you requested 20 hours ago

**Comment by TravisEz13** [comment]
@kilasuit Can you merge or rebase to resolve the test issue?

**Comment by SteveL-MSFT** [comment]
@kilasuit looks like a bad merge?  I would suggest reverting to the commit prior to pulling master in and then do something like:

```none
git pull upstream master
```

(assuming upstream points to PS/PS repo and not your fork)

**Comment by kilasuit** [comment]
Just incase others see this and fall foul to messy merges I needed to run this as my local repo's default upstream is my fork

`git pull psrepo master -s recursive -X theirs`

**Comment by vexx32** [comment]
In case you need them for future reference, these might be useful in similar situations:

- `git merge upstream master` (this one might be `upstream/master` depending on how git wants to do it? I don't recall exactly for this one)
- `git pull --rebase upstream master`
- `git rebase -i upstream/master`

(where `upstream` is the name of the remote that maps to the main PowerShell project, looks like yours was `psrepo` instead)

File: CHANGELOG.md [hunk 3]
```diff
@@ -23,6 +23,7 @@
 
 ### General Cmdlet Updates and Fixes
 
+- Add `-Emphasize` parameter to colorize `Select-String` output (#8963) (Thanks @derek-xia!)
```
Comment (SteveL-MSFT) [inline CHANGELOG.md]: Looks like you still have this commit in your PR

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

**Comment by kilasuit** [comment]
Closing this PR and opening another from a clean branch with the requested changes from this one

**Pull request closed without merging**